src/otx/algorithms/common/adapters/mmcv/configurer.py

```diff
@@ -39,30 +41,86 @@ def __init__(self, task, training):
         self.training = training
         self.ema_hooks = ["EMAHook", "CustomModelEMAHook"]  # EMA hooks supporting resume
 
-    def configure_base(self, cfg, data_cfg, data_classes, model_classes, **kwargs):
-        """Basic configuration work for recipe.
+    def configure(
+        self,
+        cfg: Config,
+        model_ckpt: str,
```

Comment:
Minor thing: how about `model_ckpt_path` instead of `model_ckpt` since it's type is `str`